src/FlagContext.ts

```diff
@@ -17,6 +17,6 @@ export interface IFlagContextValue
   >;
 }
 
-const FlagContext = React.createContext<IFlagContextValue>(null as never);
+const FlagContext = React.createContext<IFlagContextValue | null>(null);
```

Comment:
@dbasilio 

This line is a bit concerning, because if anyone is using the FlagContext directly they would receive the same compiler errors as we had in the test. Ideally, I'd love for this change to not require a major update and this is the only part that is changing the public facing API in a way that might break clients.

Reply:
I think you make a good point. And looking at our releases it's been a while since the last major. I will merge this as is and release it as 5.0.0. If we leave it I'm afraid we'll never get around to it.